pkg/controller/controller.go

```diff
@@ -1714,3 +1714,7 @@ func checkFinalizer(obj metav1.Object, finalizer string) bool {
 	}
 	return false
 }
+
+func markAsMigrated(parent context.Context) context.Context {
+	return context.WithValue(parent, connection.AdditionalInfoKey, connection.AdditionalInfo{Migrated: metrics.LabelMigrated})
```

Comment:
Here we should record the migrated status for this operation. So I think it should be
```
connection.AdditionalInfo{Migrated: <migrated-val>}
```
<migrated-val> should be pass from the caller.

For example, in CreateVolume, it should get the <migrated-val> from `result.migratedVolume`. `result` is a response we get from prepareProvisionResult.

The purpose is to record whether this operation is generated based on CSI migration.